api/system.cc

```diff
@@ -144,6 +145,25 @@ void set_system(http_context& ctx, routes& r) {
         return json::json_void();
     });
 
+    hs::set_io_limit.set(r, [&ctx](std::unique_ptr<request> req) {
+        try {
+            return ctx.db.invoke_on_all([&req] (replica::database& db) {
```

Comment:
No need to do it `_on_all()`, the seastar API is node-wide and will notify all shards as required on its own

Reply:
@xemul calling it on local() will somehow distribute the call as required?

Reply:
Yes